bitnami/etcd/templates/scripts-configmap.yaml

```diff
@@ -268,6 +269,11 @@ data:
         if etcdctl $AUTH_OPTIONS endpoint health; then
             echo "Snapshotting the keyspace..." 1>&3 2>&4
             etcdctl $AUTH_OPTIONS snapshot save "/snapshots/db" 1>&3 2>&4
+            ln /snapshots/db /snapshots/db-$(date -u "+%Y-%m-%d_%H-%M") 1>&3 2>&4
```

Comment:
Yes, I wanted to change the restore logic as little as possible. I thought making the hard-link was the best way to keep anyone relying on there being a `/snapshots/db` file happy while also not creating two completely separate files and taking twice the disk space.

Reply:
Since making the hard-link only adds new files but doesn't change the restore functionality I was more confident making this change. I can see why saving one line might be helpful. If bitnami prefers to do it without the hard link, I think I will need a bit more help to be confident that I'm not messing anyone's restore up.